protected/views/layouts/new_datasetpage.php

```diff
@@ -80,26 +80,9 @@
                                             <? } ?>
                         </ul>
                     </nav>
-                    <div class="col-xs-7 clearfix">
-                        <ul class="share-zone clearfix">
-                            <li>
-                                <a class="fa fa-facebook" style="text-decoration: none;" href="http://facebook.com/GigaScience" title="GigaScience on Facebook" aria-label="GigaScience on Facebook"></a>
-                            </li>
-                            <li>
-                                <a class="fa fa-twitter" style="text-decoration: none;" href="http://twitter.com/GigaScience" title="GigaScience on Twitter" aria-label="GigaScience on Twitter"></a>
-                            </li>
-                            <li>
-                                <a class="fa fa-weibo" style="text-decoration: none;" href="http://weibo.com/gigasciencejournal" title="Gigascience on Weibo" aria-label="GigaScience on Weibo"></a>
-                            </li>
-                            <li>
-                                <a class="fa fa-google-plus" style="text-decoration: none;" href="https://plus.google.com/u/0/104409890199786402308" title="GigaScience on Google+" aria-label="GigaScience on Google+"></a>
-                            </li>
-                            <li>
-                                <a  class="fa fa-rss" style="text-decoration: none;" href="http://gigasciencejournal.com/blog/" title="Gigascience Blog" aria-label="GigaScience Blog"></a>
-                            </li>
-                        </ul>
+                    <div class="col-xs-7 clearfix top-bar-left">
                         <div class="search-bar clearfix">
-                            <form action="/search/new" method="GET" role="search">
+                            <form action="/search/new" method="GET" role="search" class="search-form" aria-label="Datasets">
```

Comment:
This label means that the search form will be announced as `Datasets search`, please let me know if another name would be more appropriate. I chose datasets because that seems to me to be the main type of search result